modules/scf/Makefile

```diff
@@ -38,7 +38,7 @@ scf-brats-setup:
 	./brats_setup.sh
 
 .PHONY: upgrade
-upgrade:
+upgrade: chart gen-config
```

Comment:
Shall we have a helm-release target for upgrades without a chart ? (e.g. env changes, settings, etc) 
This needs doc updates https://github.com/SUSE/catapult/wiki/Running-options#upgrading-a-deployment

Reply:
I have reverted the changes to `scf-upgrade`, feels like complex enough to discuss on an issue.
I'm keeping `stratos-upgrade` as it is, it depends on `stratos-chart` and `stratos-gen-config`, so one needs to pass all env vars that one passed when deploying stratos, plus the changes that one wants.

Reply:
And by starting to write an issue on the topic, I have backpedaled and made it so one would need to do `OPTS='' make stratos-chart stratos-gen-config stratos-upgrade`, I'm a convert :stuck_out_tongue_closed_eyes: